cellxgene_schema_cli/cellxgene_schema/write_labels.py

```diff
@@ -290,16 +291,19 @@ def _add_column(self, component: str, column: str, column_definition: dict):
 
             # The sintax below is a programtic way to access obs and var in adata:
             # adata.__dict__["_obs"] is adata.obs
-            self.adata.__dict__["_" + component][new_column_name] = new_column
+            if "." in component:
+                [first_elem, second_elem] = component.split(".")
+                self.adata.__dict__["_" + first_elem].__dict__["_" + second_elem][new_column_name] = new_column
+            else:
+                self.adata.__dict__["_" + component][new_column_name] = new_column
 
```

Comment:
Could probably functionalize `.__dict__["_" + component]`, but not sure where to store that function